### PR TITLE
TKX-727 Update pipeline

### PR DESCRIPTION
**Pull request by TiagoFroisPereira**

### CODE REVIEW

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  pull-requests: write
+  contents: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Merge non major updates
+        uses: trakx/github-actions/dependabot-automerge@v10.0.9
```
Comment (emanuelgaspar) [inline .github/workflows/automerge.yml:15]:
```suggestion
        uses: trakx/github-actions/dependabot-automerge@v10.0.11
```

File: .github/workflows/nuget.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          debuggable=${debuggable:-true}
+          echo '::set-output name=debuggable::'$debuggable
+          
+          semverIncrementLevel="${{github.event.inputs.semverIncrementLevel}}"
+          semverIncrementLevel=${semverIncrementLevel:-true}
+          echo '::set-output name=semverIncrementLevel::'$semverIncrementLevel
+
+      - name: Build and publish nuget packages
+        id: publish
+        uses: trakx/github-actions/publish-nuget@v10.0.9
```
Comment (emanuelgaspar) [inline .github/workflows/nuget.yml:60]:
```suggestion
        uses: trakx/github-actions/publish-nuget@v10.0.11
```

File: .github/workflows/nuget.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Default input values
+        id: default-inputs
+        run: |
+          debuggable="${{github.event.inputs.debuggable}}"
+          debuggable=${debuggable:-true}
+          echo '::set-output name=debuggable::'$debuggable
+          
+          semverIncrementLevel="${{github.event.inputs.semverIncrementLevel}}"
+          semverIncrementLevel=${semverIncrementLevel:-true}
+          echo '::set-output name=semverIncrementLevel::'$semverIncrementLevel
```
Comment (emanuelgaspar) [inline .github/workflows/nuget.yml:56]: The `set-output` command is deprecated and will start causing errors in June this year.
Copy my updated lines from `shrimpy-api-client` or `kaiko-api-client`

**Review by emanuelgaspar — CHANGES_REQUESTED** [review]

File: .github/workflows/nuget.yml (hunk 3 above)
Comment (TiagoFroisPereira) [reply .github/workflows/nuget.yml]: Done

**Review by emanuelgaspar — APPROVED** [review]

**Review by emanuelgaspar — APPROVED** [review]

File: .github/workflows/delete.packages.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Delete packages
+      id: deletePackages
+      uses: actions/delete-package-versions@v3
+      with:
+        package-name: '${{github.event.inputs.name}}'
+        num-old-versions-to-delete: ${{github.event.inputs.num-old-versions-to-delete}}
```
Comment (monsieurleberre) [inline .github/workflows/delete.packages.yml:26]:
```suggestion
        package-type: 'nuget'
        num-old-versions-to-delete: ${{github.event.inputs.num-old-versions-to-delete}}
```

**Review by monsieurleberre — APPROVED** [review]

**Pull request merged**